Formula/c/ctpv.rb

```diff
@@ -5,6 +5,11 @@ class Ctpv < Formula
   sha256 "29e458fbc822e960f052b47a1550cb149c28768615cc2dddf21facc5c86f7463"
   license "MIT"
 
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Thank you for your contribution @suhailskhan, but the `livecheck` block isn't required here. We only use the `github_latest` strategy when necessary, because it requires use of the GitHub API, which has rate limits. In the case of `ctpv` I can't see that upstream uses GitHub pre-releases, and there are no tags present without releases, so the default `git` strategy that checks for tags should suffice.